### PR TITLE
Feature rider authentication

### PR DESCRIPTION
**Pull request by Iniyealakeret2**
1. Implemented the Splashscreen

### CODE REVIEW

**Comment by Iniyealakeret2** [comment]
2. Built UI for the welcome Screen
3. Built UI for the sign in screen

**Comment by Iniyealakeret2** [comment]
Screenshot 2022-07-14 at 16 19 36

**Comment by Iniyealakeret2** [comment]
Screenshot 2022-07-14 at 16 20 00

File: app/src/main/res/layout/fragment_login.xml [hunk 1]
```diff
@@ -0,0 +1,176 @@
...
+            <LinearLayout
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:layout_gravity="center_horizontal"
+                android:orientation="horizontal"
+                >
+                <ImageView
+                    android:layout_width="@dimen/width_42dp"
+                    android:layout_height="@dimen/height_42dp"
+                    android:layout_marginEnd="8dp"
```
Comment (IfyUnoka4) [inline app/src/main/res/layout/fragment_login.xml]: extract dimension

File: app/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -1,3 +1,16 @@
 <resources>
     <string name="app_name">Dispatch Buddy</string>
+    <!-- TODO: Remove or change this placeholder text -->
+    <string name="hello_blank_fragment">Hello blank fragment</string>
+    <string name="welcome_to_dispatchbuddy">Welcome to DispatchBuddy</string>
+    <string name="rider">Rider?</string>
+    <string name="user">User?</string>
```
Comment (IfyUnoka4) [inline app/src/main/res/values/strings.xml]: I think the User type should'nt have question mark

Comment (Iniyealakeret2) [reply app/src/main/res/values/strings.xml]: removed question marks from user types

File: app/src/main/res/layout/fragment_login.xml (hunk 1 above)
Comment (Iniyealakeret2) [reply app/src/main/res/layout/fragment_login.xml]: extracted dimension

**Pull request merged**